transformations/diacritic_removal/__init__.py

```diff
@@ -0,0 +1 @@
+from .transformation import *
```

Comment:
Please add a newline at the end of the file

Reply:
thanks, done